code/modules/robotics/bot/guardbot.dm

```diff
@@ -144,9 +144,28 @@
 											/obj/item/gun/energy/taser_gun,\
 											/obj/item/gun/energy/vuvuzela_gun,\
 											/obj/item/gun/energy/wavegun,\
-											/obj/item/gun/energy/pulse_rifle)
-
-	var/shotcount = 0		// Number of times it shoots when it should, modded by emag state
+											/obj/item/gun/energy/pulse_rifle,
+											/obj/item/gun/bling_blaster,\
+											/obj/item/bang_gun,\
+											/obj/item/gun/kinetic/meowitzer/inert,\
+											/obj/item/gun/russianrevolver,\
+											/obj/item/gun/energy/egun,\
+											/obj/item/gun/energy/crabgun,\
+											/obj/item/gun/energy/ghost,\
+											/obj/item/gun/energy/owl_safe,\
+											/obj/item/gun/energy/frog,\
+											/obj/item/gun/energy/shrinkray,\
+											/obj/item/gun/energy/glitch_gun,\
+											/obj/item/gun/energy/lawbringer)
+	// List of guns that arent wierd gimmicks or traitor weapons
+	var/global/list/budgun_actualguns = list(/obj/item/gun/energy/tasershotgun,\
+											/obj/item/gun/energy/taser_gun,\
+											/obj/item/gun/energy/wavegun,\
+											/obj/item/gun/energy/pulse_rifle,\
+											/obj/item/gun/energy/egun,\
+											/obj/item/bang_gun,\
+											/obj/item/gun/energy/lawbringer)
```

Comment:
*Personally*, the only weapons I'd have on the whitelist by default would be security equipment 🤷

Reply:
I've had reports (well, report) of people being disappointed that they can't give a Buddy a harmless gimmick gun. I feel that if someone wants to give a Buddy a dumb weapon that isn't explicitly deadly, I don't see much reason to tell them to go get an emag first.

Also apparently the crab gun is anything but harmless, so I've removed that from the list.